unstract/connectors/src/unstract/connectors/filesystems/google_drive/google_drive.py

```diff
@@ -95,3 +96,12 @@ def test_credentials(self) -> bool:
         except Exception as e:
             raise ConnectorError(str(e))
         return True
+
+    @staticmethod
+    def get_connector_root_dir(input_dir: str, **kwargs: Any) -> str:
+        """Get roor dir of gdrive."""
+        root_path = kwargs.get("root_path")
+        if root_path is None:
+            raise ValueError("root_path is required to get root_dir for Google Drive")
+        input_dir = str(Path(root_path, input_dir.lstrip("/")))
+        return f"{input_dir.strip('/')}/"
```

Comment:
Before striping and returning, can we add validations for cases where `input_dir` may not be valid?

Reply:
@harini-venkataraman `fs.isdir(input)` won't validate, afrter reported by QA. This is PR is just for making all fs connectors enable.


Let us do the enhancement once when needed. Also source.py will give the error if `fs.isdir(f"{input_dir.strip('/')}/")` is not valid